src/providers/limit-provider.js

```diff
@@ -100,9 +100,7 @@ export default class LimitProvider {
         const limits = domainObject.configuration.limits;
 
         return {
-            limits: () => {
-                return limits;
-            }
+            limits: () => Promise.resolve(limits)
```

Comment:
Nitpicky, but can we just do:
```javascript

limits: async () => { return limits; }
```

Reply:
Probably will complain it’s missing an await! Ignore me

Reply:
nitpicky but neat! I like it